docs/Deployment/realtime-analytics.md

```diff
@@ -0,0 +1,84 @@
+
+> Refer to [Analytics Realtime](https://github.com/epam/ai-dial-analytics-realtime) repository to learn more and view the project source code.
+
+Analytics Realtime tool is a sink of [vector.dev](https://vector.dev/). It does not retain any private information, such as user prompts or conversations, beyond the system. Instead, only the computed artifacts are collected and stored in time-series databases like InfluxDB or any scalable database capable of handling voluminous, constantly changing information.
+
+Examples of the computed artifacts:
+
+* Who has used the AI? – user hash, title, and never personal data such as names.
+* What areas have people asked questions about?
+* Are there any recurring patterns?
+* Topics of conversations.
+* Unique users.
+* Sentiments.
+* Cost analysis of the communication.
+* Language of conversations.
+* Any other calculated statistics based on conversations.
+
+## Configuration
+
+This section outlines the required steps for configuring Analytics Realtime service.
+
+- Step 1: Configure [DIAL Core](https://github.com/epam/ai-dial-core)
+- Step 2: Install [Influx DB](https://github.com/influxdata/influxdb)
+- Step 3: Configure [DIAL Analytics Realtime](https://github.com/epam/ai-dial-analytics-realtime)
+- Step 4: Configure [Prompt Log Collector](https://github.com/vectordotdev/vector)
+- Step 5: Configure [Grafana](https://github.com/grafana/grafana)
+
+### Step 1: AI DIAL Core
+
+Use the default AI DIAL Core [Gflog Configuration](https://github.com/epam/ai-dial-core/blob/development/src/main/resources/gflog.xml) as reference.
+
+### Step 2: Influx DB
+
+Refer to InfluxDB documentation to learn how to [install](https://docs.influxdata.com/influxdb/v2/install/) it and how to [create tokens](https://docs.influxdata.com/influxdb/v2/admin/tokens/create-token/) to read from a bucket.
+
+> Refer to [Configuration](https://github.com/epam/ai-dial-analytics-realtime?tab=readme-ov-file#configuration) to view how to configure InfluxDB for Analytics Realtime.
+
+### Step 3: AI DIAL Analytics Realtime
+
+Follow the [instructions](https://github.com/epam/ai-dial-analytics-realtime/blob/development/README.md) to setup AI DIAL Analytics Realtime service.
+
+### Step 4: Prompt Log Collector
+
+By default, AI DIAL Core uses the external open-source solution [vector.dev](https://vector.dev/) to send **chat completion logs** to storages such as AWS S3, Azure Blob Store, GCP Cloud Storage or any other "sink". It is also used to send prompt logs to AI DIAL Analytics Realtime via HTTP.
+
+> You can find more details on delivering observability data to an HTTP server in the vector.dev [documentation](https://vector.dev/docs/reference/configuration/sinks/http).
+
```

Comment:
@astsiapanay where in Analytics Realtime do we provide this configuration?

Reply:
DIAL Analytics service exposes endpoint to receive prompt logs from the collector. All what we need to know is the host name and port, e.g. http://localhost:5001/data